resources/function_help/json/$vertex_as_point

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "$vertex_as_point",
+  "type": "function",
+  "groups": ["Mesh"],
+  "description": "Returns the current vertex as a point geometry. Can be used for processing with other functions.",
```

Comment:
Not sure if "Can be used for processing with other functions." is adding any extra information - I would suggest to remove it.